content/blog/status-update-39-20.md

```diff
@@ -0,0 +1,117 @@
+---
+title: "AsyncAPI Initiative Status Update (week 39, 2020)"
+date: 2020-09-21T06:00:00+01:00
+type: blog
+featured: true
+tags:
+  - Status
+  - Community
+cover: /images/posts/status-update-39/status-update-39-cover.webp
+weight: 10
+authors:
+  - name: Lukasz Gornicki
+    photo: /images/avatars/lpgornicki.webp
+    link: https://twitter.com/derberq
+    byline: AsyncAPI Maintainer and Community Guardian
+---
+
+![cover](/images/posts/status-update-39/status-update-39-cover.webp)
+
+## Circular References Supported in HTML and Markdown Templates
+
+After recent efforts into circular references support in the AsyncAPI JavaScript Parser, now we started using these features in HTML and Markdown docs generators. With the latest releases of those two templates, you can generate documentation for schemas containing circular references. Below you can see an example specification file and how its payload and payload generated example looks like in generated HTML. You can also give it a try on your own in our [Playground](https://playground.asyncapi.io/). 
+
+<figure>
+<center>
+<img src="/images/posts/status-update-39/pic1-circ-spec.webp" alt="AsyncAPI schema with a circular reference" style="width:50%">
+<figcaption>Pic. 1: AsyncAPI schema with a circular reference</figcaption>
+</center>
+</figure> 
+
+<div class="row">
+<div class="column" class="column" style="float: left;width: 50%;">
+<center>
+<figure>
+<img src="/images/posts/status-update-39/pic2-circ-schema.webp" alt="Message payload presentation" style="width:50%">
+<figcaption>Pic. 2: Message payload presentation</figcaption>
+</figure> 
+</center>
+</div>
+<div class="column" style="float: right;width: 50%;">
+<center>
+<figure>
+<img src="/images/posts/status-update-39/pic3-circ-example.webp" alt="Generated payload example" style="width:50%">
+<figcaption>Pic. 3: Generated payload example</figcaption>
+</figure>
+</center>
+</div>
+</div> 
```

Comment:
I'd use a [custom shortcode](https://github.com/asyncapi/website#shortcodes) and try not to break mobile.

Reply:
good idea, I completely forgot about shortcodes